server/app.ts

```diff
@@ -84,6 +84,36 @@ const getUserProgressData = async (req: Request) => {
   return progress
 }
 
+const getAccountData = async (req: Request, res: Response) => {
+  if (!req.user) {
+    return res.redirect('/signin')
+  }
+  if (req.user && !req.user.acceptedPolicies) {
+    return res.redirect('/eula')
+  }
+
+  const lang = req.locale.id || 'en'
+  const translationsJSON = JSON.stringify(TRANSLATIONS[lang] || {})
+
+  const privacyPolicyMD = loadLocaleRawFile('privacy-policy.md', lang)
+
+  const userMockData = {
+    firstName: req.user?.firstName,
```

Comment:
How we are checking if the `user` exists in the first line of the method I think we can assume that the user exists at this point and remove the `?` after `req.user`.